homeworks/serhii.noskov_SerhiiNSN/GamesProject/scripts/games.js

```diff
@@ -0,0 +1,193 @@
+        release_date: '2022-07-19',
+        profile_url: 'https://www.mmobomb.com/multiversus',
+    },
+    {
+        id: 5,
+        title: 'Crossout',
+        thumbnail: 'https://www.mmobomb.com/g/5/thumbnail.jpg',
+        short_description: 'Trick out your ride and take to the post-apocalyptic roads for battle in Crossout, the free-to-play vehicular combat game from Gaijin Entertainment! Featuring a vehicle design system with endless customization and fast-paced, armor-crunching combat, Crossout offers high-octane excitement in brief and explosive matches.',
+        game_url: 'https://www.mmobomb.com/open/crossout',
+        genre: 'Shooter',
+        platform: 'PC (Windows)',
+        publisher: 'Targem',
+        developer: 'Gaijin',
+        release_date: '2017-05-30',
+        profile_url: 'https://www.mmobomb.com/crossout',
+    },
+];
+
+document.addEventListener('DOMContentLoaded', () => {
+    const template = document.querySelector('[data-type="card-template"]');
+    const checkboxNew = document.querySelector('[data-filter="new"]');
+    const checkboxOld = document.querySelector('[data-filter="old"]');
+    let gamesDisplayed = [...gamesNew];
+
+    function createCardElement(game) {
+        const cardCopy = document.createElement('li');
+        cardCopy.classList.add('games__card');
+        cardCopy.innerHTML = template.innerHTML;
+
+        cardCopy.querySelector('[data-card-img]').src = game.thumbnail;
+        cardCopy.querySelector('[data-card-title]').textContent = game.title;
+        cardCopy.querySelector('[data-card-release]').innerHTML = `<span class='bold'>Release_dat:</span> ${game.release_date}`;
+
+        return cardCopy;
+    }
+
+    function renderCards(container, games) {
+        const fragment = document.createDocumentFragment();
+        const containerElement = document.querySelector(`ul[data-type="${container}"]`);
+
+        games.forEach((game) => {
+            const card = createCardElement(game);
+            fragment.appendChild(card);
+        });
+
+        containerElement.appendChild(fragment);
+    }
+
+    function handleCheckboxChange() {
+        const isNewChecked = checkboxNew.checked;
+        const isOldChecked = checkboxOld.checked;
+
+        gamesDisplayed = gamesNew.filter((game) => {
+            const releaseYear = new Date(game.release_date).getFullYear();
+
+            if (isNewChecked && !isOldChecked) {
+                return releaseYear > 2020;
+            } if (isOldChecked && !isNewChecked) {
+                return releaseYear < 2010;
+            }
+            return true;
+        });
+
+        const cardContainer = document.querySelector('[data-type="card-container"]');
+        cardContainer.innerHTML = '';
+
+        renderCards('card-container', gamesDisplayed);
+    }
+
+    function init() {
+        checkboxNew.addEventListener('change', handleCheckboxChange);
+        checkboxOld.addEventListener('change', handleCheckboxChange);
+    }
+
+    init();
```

Comment:
This should be your starting point
> Додайте функцію init яка буде навішувати обробники подій, та відмальовувати картки без фільтації уперше, після завантаження сторінки. Це буде точка входу у ваш скріпт, те місце де все починається.

You can place this function to `'DOMContentLoaded'` handler